pkgs/tools/audio/pasystray/default.nix

```diff
@@ -5,25 +5,27 @@
 
 stdenv.mkDerivation rec {
   pname = "pasystray";
-  version = "0.7.1";
+  version = "0.8.2";
 
   src = fetchFromGitHub {
     owner = "christophgysin";
     repo = "pasystray";
-    rev = "${pname}-${version}";
-    sha256 = "0xx1bm9kimgq11a359ikabdndqg5q54pn1d1dyyjnrj0s41168fk";
+    rev = version;
+    sha256 = "sha256-QaTQ8yUviJaFEQaQm2vYAUngqHliKe8TDYqfWt1Nx/0=";
   };
 
   patches = [
-    # https://github.com/christophgysin/pasystray/issues/90#issuecomment-306190701
-    ./fix-wayland.patch
-
     # https://github.com/christophgysin/pasystray/issues/98
     (fetchpatch {
-      url = "https://sources.debian.org/data/main/p/pasystray/0.7.1-1/debian/patches/0001-Build-against-ayatana-appindicator.patch";
-      sha256 = "0hijphrf52n2zfwdnrmxlp3a7iwznnkb79awvpzplz0ia2lqywpw";
+      url = "https://sources.debian.org/data/main/p/pasystray/0.8.1-1/debian/patches/0001-Build-against-ayatana-appindicator.patch";
+      sha256 = "sha256-/HKPqVARfHr/3Vyls6a1n8ejxqW9Ztu4+8KK4jK8MkI=";
     })
-  ];
+    # https://github.com/christophgysin/pasystray/issues/90#issuecomment-361881076
```

Comment:
In addition to the links, please add a short description of the patch, just use the commit message if feeling lazy.